src/main/scala/cromwell/CommandLineParser.scala

```diff
@@ -14,8 +15,8 @@ object CommandLineParser extends App {
                                   workflowSource: Option[Path] = None,
                                   workflowInputs: Option[Path] = None,
                                   workflowOptions: Option[Path] = None,
-                                  workflowType: Option[String] = Option("WDL"),
-                                  workflowTypeVersion: Option[String] = Option("v2.0-draft"),
+                                  workflowType: Option[String] = WorkflowOptions.defaultWorkflowType,
```

Comment:
Does whatever adds the failsafe "or else just use WDL" still get applied in command line mode?

Reply:
I believe so, but will double check. Doesn't look like `MainSpec` exists anymore...

Reply:
"Failsafe" works.